src/components/Filters.js

```diff
@@ -103,10 +110,103 @@ export default class Filters {
   }
 
   reset() {
+    const rowNodes = document.querySelectorAll('[data-mt-filter-name]');
+    for (let i = 0; i < rowNodes.length; i++) {
+      rowNodes[i].parentNode.removeChild(rowNodes[i]);
+    }
     this.criteria = [];
-    this.container.innerHTML = '';
-    this.refresh();
-    this.maptable.map.reset();
+    this.maptable.render();
+  }
+
+  exportCriteria() {
```

Comment:
for all the new methods like `exportCriteria()` here and others we've added, we should add JavaDoc blocks. I know a lot of the other functions/methods haven't been documented, but we should start now and slowly work toward 100% coverage!